users/alysaght/config.h

```diff
@@ -0,0 +1,35 @@
+#define TAPPING_TERM 215
```

Comment:
```suggestion
#pragma once

#define TAPPING_TERM 215
```

Could you add a GPL2+ compatible license header here? 

For instance: 
https://github.com/qmk/qmk_firmware/blob/28ff51175b6d45ce5a8dbfe7ed6e7a12df3bc8a8/quantum/template/base/keyboard.c#L1-L15